src/class-wpml-vc-website-builder-update-translation.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * Class WPML_Beaver_Builder_Update_Translation
+ */
+class WPML_VC_Website_Builder_Update_Translation extends WPML_Page_Builders_Update_Translation {
+
+	/** @param array $data_array */
+	public function update_strings_in_modules( array &$data_array ) {
+		foreach ( $data_array['elements'] as &$element ) {
+			if ( ! in_array( $element['tag'], array( 'row', 'column' ), true ) ) {
+				$element = $this->update_strings_in_node( $element[ $this->data_settings->get_node_id_field() ], $element );
+			}
+		}
+	}
+
+	/**
+	 * @param string $node_id
+	 * @param $settings
+	 *
+	 * @return mixed
+	 */
+	public function update_strings_in_node( $node_id, $settings ) {
```

Comment:
@adrianosferreira This code is a duplicate of the code in BB.

Reply:
@brucepearson you are right. I will move it to the existing parent class.